### DISCUSSION TITLE
RollPass exceeds the maximum iteration count of 100 for all simulations.

### DISCUSSION THREAD

**Discussion by smkhalid** [pyroll-project/pyroll-pillar-model#26] [Q&A]
Hi. I am running into an issue after installing the "pyroll-piller-model" library, i am having the following Log info message while running any simulation:

"Solution iteration of RollPass 'Stand A ' exceeded the maximum iteration count of 100. Continuing anyway."

I am working in Pycharm. After adding "pyroll-piller-model" to the python interpreator, i am having the issue throughout the project. Even when i do not import the "pyroll-piller-model" in a certain simulation, the same issue occurs. 
The solution exceeds maximum iteration of 100 for any specific Roll Pass Stand and never finishes. 

I have tried it for several simulations in Projects . As soon as,  i uninstall the "pyroll-piller-model", the problem resolves.

Has anyone else come across this, or does anyone have insights into what might be causing the iterations to get stuck? Any help or suggestions would be greatly appreciated.

Thanks in advance!

**Comment by RollingTechSupport** [comment]
Please paste this in the project discussion page.
